tests/features/session/test_cookie_session.py

```diff
@@ -6,42 +6,35 @@ def test_can_start_session(self):
         request = self.make_request()
         session = self.application.make("session")
         request.cookie("s_hello", "test")
-        session.start()
+        session.start("cookie")
```

Comment:
Can you explain this change ? 🤔 I am not sure to follow.

Reply:
In this case although the "default" config for these tests is defined as "cookie" this may be changed by someone else when building other tests for other drivers.
This makes sure we are explicitly testing the "cookie" driver and not relying on the "default" which may or may not be "cookie"

Reply:
Re: the driver config ... I'm still learning the nuances of Python re passing options that have not actually assignments for the class.  
ie I assumed (incorrectly) that the class init would complain that the "timeout" and "namespace" kwargs which are not actually Redis() (or StrictRedis()) args would not work.

so yeah if this works I'm fine with it and so it will align with the other configs and reduce complexity.  

Will report back if this doesnt work as expected.

Reply:
@girardinsamuel 
after rereading your "options" config again this make much more sense... sorry its been a long day and I'm tired ;)  

so Yes the "options" at the root of the driver config makes much more sense and is a much cleaner option...
Will test this scenario in the next couple of days or so.  
Cheers for the pointers